### ISSUE TITLE
Axe accessibility test fails because of aria-required="false"

### ISSUE THREAD

**Issue by leonp5** [hackingharold/ngx-dropzone#133]
As in the heading described, the axe accessibility test fails.
No special rules:

Execution of the test:
```
it('should have no accessibility violations', async (): Promise<void> => {
expect(true).toBeTruthy();
const html = fixture.nativeElement as HTMLElement;
const results = await axe(html, (globalThis as any).axeConfig);
expect(results).toHaveNoViolations();
});
```
axe config:
```
(globalThis as any).axeConfig = {
  runOnly: {
    type: 'tag',
    values: ['wcag2a', 'wcag2aa', 'wcag2aaa'],
  },
};
```
And a simple HTML snippet inside a component

```
<ngx-mat-dropzone aria-required="true">
    <input type="file" fileInput
           [multiple]="true"
           [formControl]="formControl"
           [attr.required]="true"
           aria-required="true"
           [attr.aria-label]="'a label'"
           name="name"/>
    @for (file of files; track file.name) {
      <mat-chip-row>

        {{ file.name }}
      </mat-chip-row>
    }
  </ngx-mat-dropzone>
```
When the component is rendered, the ngx-mat-dropzone always have the aria attribute aria-required="false".
Doesn't matter, how i set it at the input or the ngx-mat-dropzone tag.
In [this Stackoverflow](https://stackoverflow.com/questions/71063218/is-there-any-benefit-to-writing-aria-required-false) post they discuss if a aria attribute is even necessary if the input isn't required.

Is there any possibility to change or remove this aria-attribute?

**Comment by hackingharold** [comment]
Hey there,

thanks for using the library. I want to make sure I understand your problem correctly.

The `aria-required` attribute on the mat-dropzone should be set automatically if you have a `Validators.required` set on your `formControl`. If you want to override it manually, try adding brackets `[aria-required]`. Having the attribute on the `<input type="file" />` element should not be necessary.

If this does not help, could you share the exact error message? I have not been using axe yet.

**Comment by leonp5** [comment]
@hackingharold thanks for your response 👍 

Here the axe error message:

```
Error: expect(received).toHaveNoViolations(expected)

Expected the HTML found at $('#mat-dropzone-component-1') to have no violations:

<ngx-mat-dropzone tabindex="0" ondragover="event.preventDefault()" class="ng-tns-c508571215-0 ng-untouched ng-pristine ng-valid" style="width: 0px; margin-top: -16px; margin-bottom: -16px;" aria-describedby="mat-mdc-hint-a4 mat-mdc-hint-a5" aria-invalid="false" id="mat-dropzone-component-1" aria-required="false">

Received:

"Elements must only use supported ARIA attributes (aria-allowed-attr)"

Fix all of the following:
  ARIA attribute is not allowed: aria-required="false"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI
```

I understand axe in this way, that aria-required="false" doesn't make any sense. So if the input isn't required there shouldn't be an aria-attribute which "only" says that the input isn't required (that's what i also found in the above linked Stackoverflow post).

So if the input isn't required, the aria-attribute should be removed. But this isn't possible at the moment.

**Comment by hackingharold** [comment]
I just published v19.2.1 which should remove the `aria-required` attribute from the DOM instead of setting it to false.
The attribute should only be set when true. Please check it out.

**Comment by leonp5** [comment]
That's great. It's fixed now. Thank you very much 🙂
